app/routes/administrative-units/administrative-unit/governing-bodies/governing-body/index.js

```diff
@@ -1,29 +1,47 @@
 import Route from '@ember/routing/route';
+import { inject as service } from '@ember/service';
 
 const MEMBER_ROLE_ID = '2e021095727b2464459a63e16ebeafd2';
 
 export default class AdministrativeUnitsAdministrativeUnitGoverningBodiesGoverningBodyIndexRoute extends Route {
+  @service store;
+
   async model() {
     let { administrativeUnit, governingBodyClassification, governingBody } =
       this.modelFor(
         'administrative-units.administrative-unit.governing-bodies.governing-body'
       );
 
-    let mandates = await governingBody.mandates;
-    let memberMandates = mandates.filter((mandate) => {
-      return mandate.get('roleBoard.id') === MEMBER_ROLE_ID;
+    let governingBodyMandatories = await this.store.query('worship-mandatory', {
```

Comment:
Hmm, the current version will be limited to 20 records since that's the default. And pagination won't work as expected since we split the array manually. Ideally we could just do 2 fetches where the filtering is done server side, but `NOT` checks aren't supported last time I checked.

Reply:
Can't you edit the page size in the query?

Reply:
From what I've seen in the data, 20 records will be enough anyway, but it's good to keep this in mind or add a comment.

Reply:
Yea, I was trying to make the `not` version work with a `:gt:`  and `:lt:` filter hack. but that doesn't seem to work for ids.

Reply:
A `not` filter would be real handy though.

Reply:
I solved it by including all the ids (besides the lid one) :smile:, best hacky workaround I could find.